Packs/Axonius/Integrations/Axonius/Axonius.py

```diff
@@ -0,0 +1,213 @@
+"""Demisto Integration for Axonius."""
+import traceback
+from typing import Any, List, Optional, Union
+
+import demistomock as demisto
+from axonius_api_client.api.assets.devices import Devices
+from axonius_api_client.api.assets.users import Users
+from axonius_api_client.connect import Connect
+from axonius_api_client.tools import dt_parse, strip_left
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+MAX_ROWS: int = 50
+"""Maximum number of assets to allow user to fetch."""
+SKIPS: List[str] = ["specific_data.data.image"]
+"""Fields to remove from each asset if found."""
+FIELDS_TIME: List[str] = ["seen", "fetch", "time", "date"]
+"""Fields to try and convert to date time if they have these words in them."""
+
+
+def get_int_arg(
+    key: str, required: Optional[bool] = False, default: Optional[int] = None,
+) -> int:
+    """Get a key from a command arg and convert it into an int."""
+    args: dict = demisto.args()
+    value: int = args.get(key, default)
+
+    if value is None and required:
+        raise ValueError(f"No value supplied for argument {key!r}")
+
+    try:
+        return int(value)
+    except Exception:
+        raise ValueError(
+            f"Supplied value {value!r} for argument {key!r} is not an integer."
+        )
+
+
+def get_csv_arg(
```

Comment:
You can use `argToList` function from `CommonServerPython` imported at the beginning of this py file.

Reply:
@guyfreund Updated to use argToList in our function while still checking for empty list items. The argToList function has a short coming in that it does not remove "empty" strings produced by input like "a,b,,c,d". We coded around that.

Reply:
OK, whatever is good for you